IMC/chatbot/Chatbot.md

```diff
@@ -0,0 +1,170 @@
+# Covid-19 and ChatBots
```

Comment:
Create a different title for the tutorial like: 
"Creating a Chatbot for COVID-19 Data"

Reply:
I change the title to "Creating a Chatbot for COVID-19"